mockwebserver/src/main/java/okhttp3/mockwebserver/RecordedRequest.kt

```diff
@@ -105,12 +115,16 @@ class RecordedRequest(
     }
   }
 
+  @Deprecated(
+      message = "Use body.readUtf8()",
+      replaceWith = ReplaceWith("body.readUtf8()"),
```

Comment:
curious: Does Intellij also convert the kotlin suggestions to Java?

Reply:
No, Java users have to fix manually.